src/main/kotlin/io/github/enyason/predictions/models/Prediction.kt

```diff
@@ -0,0 +1,23 @@
+package io.github.enyason.predictions.models
+
+import com.google.gson.annotations.SerializedName
+
+data class Prediction(
+    var id: String? = null,
+    var model: String? = null,
+    var version: String? = null,
+    var input: Input? = null,
+    var logs: String? = null,
+    var output: String? = null,
```

Comment:
Replicate has more than one representation for output. see here

https://replicate.com/anotherjesse/zeroscope-v2-xl?output=json

https://replicate.com/fofr/sdxl-emoji?output=json

https://replicate.com/fofr/sdxl-emoji?output=json


so far, i see it's wither a string or a list